ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/prompt.j2

```diff
@@ -0,0 +1,5 @@
+{# cli prompt #}
+!
+{% if prompt is defined and prompt is not none %}
```

Comment:
please ues the new arsita.avd.defined filter, which also handles check for if not none/null value.

```jinja
{% if prompt is arista.avd.defined %}
```